### ISSUE TITLE
Jupyterlab not starting after upgrade to > 2.

### ISSUE THREAD

**Issue by statiksof** [jupyterlab/jupyterlab#8773]
## Description

when I upgrade to any version > 2. the JupyterLab notebook does not start.
Until 1.2.16 it worked fine, I am using `jupyter-labhub` within `Jupyterhub`. 

This is error I see:

```
from jupyterlab.labhubapp import main
ModuleNotFoundError: No module named 'jupyterlab.labhubapp'
```

## Context

- Jupyterhub 1.1.0:
- JupyterLab version: 2.2.2 (I tried all 2.xxx)

**Comment by ricardoprins** [comment]
Same issue here.

**Comment by statiksof** [comment]
I noticed that the after the upgrade, ```jupyter-labhub``` is not in the ```bin```, this is why the script didn't find the binary. 
I am not sure if installing with pip will change this behavior.

**Comment by statiksof** [comment]
@ricardoprins I think we are going to answer our self here :smile: 

Actually, ```jupyter-labhub``` has been removed since ```2.0.X```. So you can change the command to ```jupyterhub-singleuser```.
This worked for me.

**Comment by blink1073** [comment]
We had already planned to bring back the `labhubapp` in 3.0, so I opened https://github.com/jupyterlab/jupyterlab/pull/8780 to restore it in 2.2

**Comment by blink1073** [comment]
Fixed in JupyterLab 2.2.4
